src/services/CertificateGenerationService.ts

```diff
@@ -507,9 +544,48 @@ class CertificateGenerationService {
                 };
                 console.log("CHECK HERE DOCGENPAYLOAD -> ", docGenPayloadAdr);
                 return docGenPayloadAdr;
+            case CERTIFICATE_DATA.IVA_DATA:
+                const ivaFailDetailsForDocGen = {
+                    SerialNumber: testResult.vehicleType === "trl" ? testResult.trailerId : testResult.vrm,
+                    VehicleTrailerNrNo: testResult.vehicleType === "trl" ? testResult.trailerId : testResult.vrm,
+                    TestCategoryClass: testResult.euVehicleCategory,
+                    TestCategoryBasicNormal: this.isBasicIvaTest(testResult.testTypes.testTypeId) ? IVA_30.BASIC : IVA_30.NORMAL,
+                    Make: testResult.make,
+                    Model: testResult.model,
+                    BodyType: testResult.bodyType,
+                    Date: moment(testResult.testTypes.createdAt).format("DD.MM.YYYY"),
+                    ReapplicationDate: moment(testResult.testTypes.createdAt)
+                        .add(6, "months")
+                        .subtract(1, "day")
+                        .format("DD.MM.YYYY"),
+                    Station: testResult.testStationName,
+                    AdditionalDefects:
+                        testResult.testTypes.customDefects && testResult.testTypes.customDefects.length > 0
+                            ? testResult.testTypes.customDefects
+                            : IVA_30.EMPTY_CUSTOM_DEFECTS,
+                    IvaDefects: testResult.testTypes.ivaDefects,
+                };
+                console.log("CHECK HERE DOCGENPAYLOAD -> ", ivaFailDetailsForDocGen);
```

Comment:
Is this another debug log?

Reply:
I think it may be a good idea to leave this in until the iva-30-front page and doc-gen tickets have been completed. We can remove any unwanted loggers as part of the holding branch merge into develop.